WordPress/src/main/java/org/wordpress/android/ui/reader/SubfilterBottomSheetFragment.kt

```diff
@@ -129,6 +130,10 @@ class SubfilterBottomSheetFragment : BottomSheetDialogFragment() {
                 behavior.peekHeight = metrics.heightPixels / 2
             }
         }
+
+        viewModel.isTitleContainerVisible.observe(viewLifecycleOwner) { isVisible ->
+            titleContainer.visibility = if (isVisible) View.VISIBLE else View.GONE
```

Comment:
Nitpick: Would it make sense to use the `View.setVisible` extension function?

```
titleContainer.setVisible(isVisible)
```

Reply:
Definitely, not sure why I didn't use it, I even called the variable `isVisible`... 🤦🏼